text/0000-router-helpers.md

```diff
@@ -0,0 +1,415 @@
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- RFC PR: (leave this empty)
+- Ember Issue: (leave this empty)
+
+# Router Helpers & Modifiers
+
+## Summary
+
+This RFC introduces new router helpers that represent a decomposition of functionality of what we commonly use `{{link-to}}` for. This RFC also deprecates the `{{link-to}}` component, `{{query-params}}` helper, and default query param serialization. Below is a list of the new helpers:
+
+```hbs
+{{transition-to}}
+{{url-for}}
```

Comment:
url-for vs href-for?

Reply:
These are more less intended declarative APIs that map directly on to the router service methods. Changing the name would introduce some incoherence here.